docs/src/man/hosting.md

```diff
@@ -128,9 +128,11 @@ jobs:
       julia: 1.0
       os: linux
       script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()));
-                                               Pkg.instantiate()'
-        - julia --project=docs/ docs/make.jl
+         - julia -e 'using Pkg; Pkg.activate(); push!(LOAD_PATH, pwd());' 
+        - julia --project=docs/ -e 'using Pkg; Pkg.activate();  push!(LOAD_PATH,pwd());'
+
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate();'
+        - julia --project=docs/ --color=yes docs/make.jl
```

Comment:
The `push!`es to `LOAD_PATH` have no effect and are unnecessary. What is your goal here?

Reply:
Travis gave me errors in the documentation stage when I used the original config:
https://travis-ci.com/juliamatlab/MatLang/jobs/232115453/config

But I fixed the issue by adding these two lines:
https://travis-ci.com/juliamatlab/MatLang/jobs/232117413/config

I think it was because my package is not registered. 

I got those errors when building the package offline in Windows using PowerShell.